app/views/static/evacuate-form.html.haml

```diff
@@ -0,0 +1,59 @@
+%form#form_div{:name           => "angularForm",
+               "miq-form"      => true,
+               'form-changed'  => true,
+               "model"         => "vm.vmCloudModel",
+               "model-copy"    => 'vm.modelCopy'}
+  = render :partial => "layouts/flash_msg"
+  %h3
+    = _('Evacuate Host')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Auto-select Host?')
+      .col-md-8
+        %input{:type         => "checkbox",
+               :name         => "auto_select_host",
+               'ng-model'    => "vm.vmCloudModel.auto_select_host",
+               :required     => true,
+               'ng-disabled' => "vm.hosts.length == 0"}
+
+        %input{:type         => "hidden",
+               :name         => "auto_select_host",
+               :value        => "on",
+               'ng-if'       => "vm.hosts.length == 0"}
+    .form-group
+      %label.col-md-2.control-label
+        = _('On Shared Storage')
+      .col-md-8
+        %input{:type         => "checkbox",
+               :name         => "on_shared_storage",
+               'ng-model'    => "vm.vmCloudModel.on_shared_storage"}
+    .form-group{'ng-hide' => 'vm.vmCloudModel.on_shared_storage'}
+      %label.col-md-2.control-label
+        = _('Admin Password')
+      .col-md-8
+        %input.form-control{:type         => "password",
+                            :name         => "admin_password",
+                            'ng-model'    => "vm.vmCloudModel.admin_password"}
+  #evacuate-select-destination-host{'ng-hide' => 'vm.vmCloudModel.auto_select_host'}
+    %h3
+      = _('Select Destination Host')
+    .form-horizontal
+      .form-group
+        %label.col-md-2.control-label
+          = _('Destination Host')
+        .col-md-8
+          %select{:name        => 'destination_host',
+                  'ng-model'   => 'vm.vmCloudModel.host',
+                  'ng-options' => 'host.name as host.name for host in vm.hosts track by host.id'}
+
+  %div_for_paging{'ng-controller'                    => "pagingDivButtonGroupController",
```

Comment:
is this being used?

Reply:
I don't think so.

Reply:
Nope this is needed to get buttons in correct place.